### PR TITLE
union pars scripts

### PR DESCRIPTION
**Pull request by S-txt**

### CODE REVIEW

File: parse.py [hunk 1]
```diff
@@ -19,13 +19,35 @@
 reply = vk.wall.get(owner_id=owner_id, count=0)
 total_count = reply['count']
 
+newDB = false
```
Comment (S-txt) [inline parse.py]: newDB = true

File: parse.py [hunk 2]
```diff
@@ -19,13 +19,35 @@
...
+old_total_count = 0
+data = None
+# >>> Мы ПЫТАЕМСЯ прочитать файл с данными
+try:
+  with open('posts.json', 'r', encoding='utf-8') as outfile:
+    data = json.load(outfile)
+    old_total_count = data['total_count']
+    # FIXME: Костыль (возможно)
+    newDB = true
```
Comment (S-txt) [inline parse.py]: newDB = false

File: parse.py [hunk 3]
```diff
@@ -19,13 +19,35 @@
...
+    # считаем, сколько постов добавилось, и задаем max_count
+    count = total_count - old_total_count
+    max_count = count // 25 + 1
+
 # парсим посты и переворачиваем список с ними, потому что нам нужны данные от старых к новым, а парсятся
 # от новых к старым
+# также обрезаем список до нужного нам кол-ва постов, так как get_all получает кол-во постов, кратное 25
 reply = tools.get_all(method='wall.get', max_count=1, values={'owner_id': owner_id}, limit=1)
+items = reply['items'][:count:]
 items = reply['items'][::-1]
```
Comment (SofyaAksenova) [inline parse.py]: items = items[::-1]

File: parse.py [hunk 4]
```diff
@@ -19,13 +19,35 @@
...
+finally:
+    # >>> По окончанию блока подставляем данные о тотале
+    # считаем, сколько постов добавилось, и задаем max_count
+    count = total_count - old_total_count
+    max_count = count // 25 + 1
+
 # парсим посты и переворачиваем список с ними, потому что нам нужны данные от старых к новым, а парсятся
 # от новых к старым
+# также обрезаем список до нужного нам кол-ва постов, так как get_all получает кол-во постов, кратное 25
 reply = tools.get_all(method='wall.get', max_count=1, values={'owner_id': owner_id}, limit=1)
```
Comment (SofyaAksenova) [inline parse.py]: max_count=max_count

**Pull request closed without merging**